DDDEastAnglia/Models/SponsorModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DDDEastAnglia.Models
+{
+    public class SponsorModel
```

Comment:
I'd like some guidence on how we pass data from UI to DB and back again..

What are the models we use etc..